src/main/resources/org/biouno/unochoice/stapler/unochoice/UnoChoice.es6

```diff
@@ -694,14 +705,15 @@ var UnoChoice = UnoChoice || (jQuery3 => {
                             input.checked = false;
                             let jsonInput = util.makeHidden(input.getAttribute('otherid'), input.getAttribute('json'), '', input.getAttribute('value'), input.getAttribute('name'), input.getAttribute('alt'));
 
-                            let label = util.makeLabel(input.getAttribute('alt'), undefined);
+                            let label = util.makeLabel(input.getAttribute('alt'), undefined, "jenkins-radio__label");
```

Comment:
more or less the same question: Can `entry` here ever be something else than an input of type `radio`

PS: line 703 is obsolete as `input` is replaced by `entry` in line 704